### PR TITLE
Self destruct. (FStorage bus, FImport/Export bus, and more)

### PR DESCRIPTION
**Pull request by firenoo**
Should AE2FC be detected, the following CABLE parts will replace themselves:
* Fluid Storage Bus
* Fluid Import Bus
* Fluid Export Bus
* Fluid Level Emitter
* Fluid Storage Monitor
* Fluid Conversion Monitor
* Fluid Interface -> AE2FC Dual Interface

~~Depends on https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/276~~

~~If that PR is not merged, this PR will do nothing.~~ Done, thanks

### CODE REVIEW

**Comment by Dream-Master** [comment]
yea will add after stable

File: src/main/scala/extracells/part/PartFluidStorageMonitor.java [hunk 1]
```diff
@@ -442,4 +447,34 @@ public void writeToStream(ByteBuf data) throws IOException {
...
+            // Fluid
+            // long amount = extra.getLong("amount");
+            int fluid = extra.getInteger("fluid");
+            Fluid f = FluidRegistry.getFluid(fluid);
+            NBTTagCompound itemDisplay = FluidCraft.createFluidNBT(f.getName(), 0L);
+            extra.setTag("configuredItem", itemDisplay);
+            extra.removeTag("amount");
+            extra.removeTag("fluid");
+            // Locked and other props
+            extra.setBoolean("isLocked", extra.getBoolean("locked"));
```
Comment (OneEyeMaker) [inline src/main/scala/extracells/part/PartFluidStorageMonitor.java:471]:
```suggestion
            extra.setBoolean("isLocked", extra.getBoolean("locked"));
            extra.removeTag("locked");
```

File: src/main/scala/extracells/part/PartFluidStorage.java [hunk 2]
```diff
@@ -318,4 +321,66 @@ private boolean wasChanged() {
...
+				0,  9, 18, 27, 36, 45,
+				1, 10, 19, 28, 37, 46,
+				2, 11, 20, 29, 38, 47,
+				3, 12, 21, 30, 39, 48,
+				4, 13, 22, 31, 40, 49,
+				5, 14, 23, 32, 41, 50,
+				6, 15, 24, 33, 42, 51,
+				7, 16, 25, 34, 43, 52,
+				8, 17, 26, 35, 44, 53 };
+			//spotless:on
```
Comment (OneEyeMaker) [inline src/main/scala/extracells/part/PartFluidStorage.java]: Manual spotless.

```suggestion
            // spotless:off
            FILTER_MAP = new int[] {
                0,  9, 18, 27, 36, 45,
                1, 10, 19, 28, 37, 46,
                2, 11, 20, 29, 38, 47,
                3, 12, 21, 30, 39, 48,
                4, 13, 22, 31, 40, 49,
                5, 14, 23, 32, 41, 50,
                6, 15, 24, 33, 42, 51,
                7, 16, 25, 34, 43, 52,
                8, 17, 26, 35, 44, 53 };
            //spotless:on
```

Comment (firenoo) [reply src/main/scala/extracells/part/PartFluidStorage.java]: Sorry, I blame IntelliJ Scala Plugin for this one... made my tab settings wonky.

File: src/main/scala/extracells/part/PartFluidStorageMonitor.java (hunk 1 above)
Comment (firenoo) [reply src/main/scala/extracells/part/PartFluidStorageMonitor.java:471]: Done.

**Comment by firenoo** [comment]
Seems like its broken on the full pack. Investigating.

**Comment by firenoo** [comment]
Cleaned up the commits and is now ready. Did quite a bit of testing on the full pack.

**Review by OneEyeMaker — APPROVED** [review]

**Pull request merged**